### PR TITLE
DOC: Specify benchmark dataset size in readme and index.rst

### PR DESCRIPTION
**Pull request by RichardScottOZ**
or something like that, so it can be seen straight away?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,6 @@ as possible:
 
 The following chart gives an impression of the speed improvement that can be expected
 when processing larger files. The [benchmarks](https://github.com/geofileops/geobenchmark)
-ran on a Windows PC with 12 cores and include I/O.
+ran on a Windows PC with 12 cores and include I/O.  The benchmark dataset was 500K polygons.
```
Comment (theroggy) [inline README.md:36]:
```suggestion
typically use input file(s) with 500K polygons, ran on a Windows PC with 12 cores and include I/O.
```

Comment (theroggy) [reply README.md:36]: True, is indeed important info to be able to interprete the timings.

Not sure what is best/clearest, but I tried to clarify that for most 2-layer operations (intersection, union) both input layers are 500k polygons. 

Note that the symmetrical difference benchmark has one 500k input layer, and one with only 10 features, but quite complex ones (20.000 points per polygon).

**Comment by theroggy** [comment]
Onze the exact wording is clear... the same sentence is repeated in the readthedocs landing page: https://github.com/geofileops/geofileops/blob/main/docs/index.rst

File: README.md (hunk 1 above)
Comment (RichardScottOZ) [reply README.md:36]: Makes sense.

**Pull request merged**